bin/integration-cleanup

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+set -e
+
+
+# Age tolerance threshold for running instances (in seconds); defaults to 20 minutes.
+#   Override in CircleCI by defining INSTANCE_AGE_THRESHOLD for .env.integration.
+AGE_THRESHOLD=${INSTANCE_AGE_THRESHOLD:-"1200"}
```

Comment:
I'd suggest using something like four hours for safety.  Integration tests can run concurrently on Circle CI, and deployments take longer than 20 minutes, so with a threshold this low we might kill VMs belonging to concurrent test runs.